CHANGES.md

```diff
@@ -12,6 +12,10 @@ $ towncrier create <PR-number>.<break|feat|fix>.md --content "Short description"
 
 <!-- towncrier release notes start -->
 
+### Enhancements
+
+[#493](https://github.com/cylc/cylc-uiserver/pull/493) - Added mutation for scanning filesystem.
+
```

Comment:
```suggestion
```
We have switched to a new way of generating the changelog (see the comment above). But actually this change does not need a changelog entry as no-one is going to use the mutation directly (not to say it isn't an important change though!)